daml-lf/archive/BUILD.bazel

```diff
@@ -151,6 +152,14 @@ java_proto_library(
     deps = [":daml_lf_proto_lib"],
 )
 
+javadoc_library(
+    name = "daml_lf_archive_javadoc",
```

Comment:
Considering that the bazel target in artifacts.yaml is called `daml_lf_archive_java`, should this be called `daml_lf_archive_java_javadoc`?

Reply:
Good catch, fixed! (I’ll add the check before attempting to make another release).